staging/src/k8s.io/client-go/tools/metrics/metrics.go

```diff
@@ -37,21 +47,51 @@ type ResultMetric interface {
 }
 
 var (
+	// ClientCertExpiration is the lifetime of the client certificate. The value measured in seconds since January 1, 1970 UTC.
+	ClientCertExpiration ExpirationMetric = noopExpiration{}
```

Comment:
This comment should probably be updated.

Reply:
Done.